src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/service/ServiceCategoryServiceTest.kt

```diff
@@ -7,23 +7,33 @@ import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager
 import org.springframework.test.context.ActiveProfiles
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceCategory
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository.ServiceCategoryRepository
+import java.time.OffsetDateTime
+import java.util.UUID
 
 @DataJpaTest
-@ActiveProfiles("db")
+@ActiveProfiles("jpa-test")
 class ServiceCategoryServiceTest @Autowired constructor(
   val entityManager: TestEntityManager,
   val serviceCategoryRepository: ServiceCategoryRepository
 ) {
-
   private val serviceCategoryService = ServiceCategoryService(serviceCategoryRepository)
 
   @Test
-  fun `get all serviceCategories`() {
-    val categories = listOf(ServiceCategory(name = "Category 1"), ServiceCategory(name = "Category 2"))
+  fun `get non-existent service category returns null`() {
+    assertThat(serviceCategoryService.getServiceCategoryByID(UUID.randomUUID())).isNull()
+  }
+
+  @Test
+  fun `get valid service category`() {
+    val idToFind = UUID.randomUUID()
+
+    val categories = listOf(
+      ServiceCategory(name = "Accomodation", id = idToFind, created = OffsetDateTime.now(), complexityLevels = emptyList()),
+      ServiceCategory(name = "Accomodation", id = UUID.randomUUID(), created = OffsetDateTime.now(), complexityLevels = emptyList())
+    )
     categories.forEach { entityManager.persist(it) }
```

Comment:
This can be converted to a .map() on the list which removes the need for the 'categories' variable